src/config.c

```diff
@@ -2506,6 +2506,23 @@ int updateRequirePass(sds val, sds prev, const char **err) {
     return 1;
 }
 
+
+int updateClusterFlags(int val, int prev, const char **err) {
+    UNUSED(val);
+    UNUSED(prev);
+    UNUSED(err);
+    clusterUpdateMyselfFlags();
+    return 1;
+}
+
+int updateClusterIp(char *val, char *prev, const char **err) {
+    UNUSED(val);
+    UNUSED(prev);
+    UNUSED(err);
+    clusterUpdateMyselfIp();
```

Comment:
If we're going to do this we might as well try to respect API boundaries and pass through the new value.

Reply:
You mean call like `clusterUpdateMyselfIp(val)` ?

Reply:
yes